runtime/include/vendor_version.h

```diff
@@ -21,7 +21,7 @@
  *******************************************************************************/
 
 /* This file provides a means to supply vendor specific version info such as 
```

Comment:
Can you remove the trailing whitespace while you're here, please.

Reply:
Yes, but perhaps you should just remove it from every file in one go so we can be done with this in existing code.

Reply:
Pushed.

Reply:
Sadly, trailing whitespace seems to come back quite quickly: I think we would need to augment the 'line-end check' to avoid that.